website_event_sale_hide_ticket/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Website Event Sale Hide Ticket",
+    "summary": "Allow to hide event ticket from the website",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
```

Comment:
Still in alpha?

Reply:
changed to beta but not stable as not used in production yet